tests/saw/spec/handshake/rfc_handshake_tls13.cry

```diff
@@ -305,3 +305,61 @@ rfc2S2N msg = mkAct recordType messageType writer
                  | msg.sender == both then 'B'
                  else '!'
         noMessageType = TLS_HELLO_REQUEST // HelloRequests don't exist in tls1.3
+
+/**
+ * printHandshake and s2nToWords are two helper functions for debugging cryptol
+ * Take the counterexample from tls13rfcSimulatesS2N and plug the values into
```

Comment:
When you say "take the counter example", do you mean literally plug those values into this Cryptol file's testParameters definition?

Reply:
yes. parameter and connection values